ephemeris/shed_install.py

```diff
@@ -225,122 +139,128 @@ def installed_tool_revisions(gi=None, omit=None):
             skip = False
             # Check if we already processed this tool and, if so, add the new
             # revision to the existing list entry
-            for ir in installed_revisions_list:
-                if the_same_tool(it, ir):
-                    ir['revisions'].append(it.get('changeset_revision', None))
+            for installed_revision in installed_revisions_list:
+                if the_same_repository(installed_repository, installed_revision):
+                    installed_revision['revisions'].append(installed_repository.get('changeset_revision', None))
                     skip = True
             # Check if the repo name is contained in the 'omit' list
-            for o in omit:
-                if o in it['name']:
+            for omitted_repository in omit:
+                if omitted_repository in installed_repository['name']:
                     skip = True
             # We have not processed this tool so create a list entry
             if not skip:
-                ti = {'name': it['name'],
-                      'owner': it['owner'],
-                      'revisions': [it.get('changeset_revision', None)],
-                      'tool_shed_url': 'https://' + it['tool_shed']}
-                installed_revisions_list.append(ti)
+                repo_info = {
+                    'name': installed_repository['name'],
+                    'owner': installed_repository['owner'],
+                    'revisions': [installed_repository.get('changeset_revision', None)],
+                    'tool_shed_url': 'https://' + installed_repository['tool_shed'],
+                }
+                installed_revisions_list.append(repo_info)
     return installed_revisions_list
 
 
-def installed_tools(gi, omit=None):
+def installed_repositories(gi, omit=None):
     """
-    Get a list of tools on a Galaxy instance.
+    Get a list of repositories on a Galaxy instance.
     :type gi: GalaxyInstance object
     :param gi: A GalaxyInstance object as retured by `galaxy_instance` method.
     :type omit: list of strings
-    :param omit: A list of strings that, if found in a tool name, will result
-                    in the tool not being included in the returned list.
+    :param omit: A list of strings that, if found in a repository name, will result
+                    in the repository not being included in the returned list.
     :rtype: dict
     :return: The returned dictionary contains the following keys, each
              containing a list of dictionaries:
-                - `tool_panel_shed_tools` with a list of tools available in the
+                - `tool_panel_shed_repositories` with a list of repositories available in the
                 tool panel that were installed on the target Galaxy instance
                 from the Tool Shed;
-                - `tool_panel_custom_tools` with a list of tools available in
+                - `tool_panel_custom_tools` with a list of repositories available in
```

Comment:
repositories should here be tool, right?

Reply:
Yes. I did some find -> replace (in selection) stuff. It worked better than just doing it for the entire file. But I still missed some things. Thanks for pointing them out.